packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -475,6 +475,21 @@ export default class ConversationService {
     });
   }
 
+  public async addUser(conversationId: string, userId: string): Promise<string[]>;
+  public async addUser(conversationId: string, userIds: string[]): Promise<string[]>;
+  public async addUser(conversationId: string, userIds: string | string[]): Promise<string[]> {
```

Comment:
Isn't this a duplication?

Reply:
Usually that's what [function overloading](https://www.typescriptlang.org/docs/handbook/functions.html) looks like but in this case it's duplication because they all return the same.

Reply:
So yes, I suggest to remove the overloading.

Reply:
Ok, I will make `addUser` return just a single `string` then.